.changeset/long-bottles-brush.md

```diff
@@ -0,0 +1,23 @@
+---
+"@comet/cms-api": major
+---
+
+Rename DateFilter to DateTimeFilter
+
+This leaves room for a future DateFilter that only filters by date, not time.
+
+## Upgrading
+
+1. Replace
+
+```ts
+import { DateFilter } from "@comet/cms-api";
+```
+
+with
+
+```ts
+import { DateTimeFilter } from "@comet/cms-api";
+```
```

Comment:
Optional: Could also be done with a diff:

```
1. Update import

``diff
- import { DateFilter } from "@comet/cms-api";
+ import { DateTimeFilter } from "@comet/cms-api";
``
```

Reply:
09d0b8e2dff80462070e7802c6ab8dfa0cc8e5f4